e2e/__tests__/__snapshots__/coverage_report.test.js.snap

```diff
@@ -46,16 +46,16 @@ exports[`generates coverage when using the testRegex config param  1`] = `
 "-------------------------------------|----------|----------|----------|----------|-------------------|
 File                                 |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
 -------------------------------------|----------|----------|----------|----------|-------------------|
-All files                           </> |    56.52</> |        0</> |       50</> |    55.56</> |                  </> |
- coverage-report                    </> |    41.18</> |        0</> |       25</> |    42.86</> |                  </> |
-  OtherFile.js                      </> |      100</> |      100</> |      100</> |      100</> |                  </> |
-  Sum.js                            </> |    85.71</> |      100</> |       50</> |      100</> |                  </> |
-  SumDependency.js                  </> |        0</> |        0</> |        0</> |        0</> |           8,10,12</> |
-  notRequiredInTestSuite.js         </> |        0</> |        0</> |        0</> |        0</> |     8,15,16,17,19</> |
- coverage-report/cached-duplicates/a</> |      100</> |      100</> |      100</> |      100</> |                  </> |
-  Identical.js                      </> |      100</> |      100</> |      100</> |      100</> |                  </> |
- coverage-report/cached-duplicates/b</> |      100</> |      100</> |      100</> |      100</> |                  </> |
-  Identical.js                      </> |      100</> |      100</> |      100</> |      100</> |                  </> |
+All files                            |    56.52 |        0 |       50 |    55.56 |                   |
```

Comment:
@thymikee any idea what could cause the resets to be gone?

Reply:
It makes the snapshot easier to read as the columns match up, but...

Reply:
Nope, but maybe it's the same thing that happens to @cpojer in reporter?